docs/changes.rst

```diff
@@ -22,6 +22,15 @@ You should generally assume that an API is internal unless you have specific
 information to the contrary.
 
 -------------------
+3.11.3 - 2017-06-10
```

Comment:
I think this bound is wrong. [`max\_digits` includes the digits after the decimal place](https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.DecimalField.max_digits). Of course, that means the previous version was *also* wrong.